client/trino-jdbc/pom.xml

```diff
@@ -404,6 +404,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <jdbc-jar>${project.build.directory}/${project.name}-${project.version}.jar</jdbc-jar>
```

Comment:
This won't work after a clean build, since tests run in the `test` phase which [happens before](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#default-lifecycle) the `package` phase where the JAR is built. 

We could do this as an integration test using `maven-failsafe-plugin`. See the `trino-server-rpm` module for an example where we do a similar test for the RPM.

Reply:
Correct. Thanks for the pointer!